services/py-api/src/server/handlers/participants_handlers.py

```diff
@@ -37,3 +43,10 @@ async def create_participant(
             return ErrResponse(error="An unexpected error occurred during the creation of Participant")
 
         return ParticipantRegisteredInTeamResponse(participant=result.ok_value[0], team=result.ok_value[1])
+
+
+    async def capacity_reached(self):
+        return JSONResponse.error(
+            message = "The hackathon capacity has been reached",
+            status_code = 409
+        )
```

Comment:
Why would you have an async method to return a JSONResponse error in the handler method. If you would ever need to return a JSONResponse just use the class provided by the library no need to create a wrapper method around it. This method does not belong here, so I suggest deleting it.

Reply:
Also we never return JSONresponse objects. Haven't you looked at the `participants_hablder.py`?

Reply:
Fixed in 1bec98abe406eea6382a5453ac98f3e900d3ddd4